cpu/native/include/can_params.h

```diff
@@ -30,8 +30,12 @@ extern "C" {
  * @brief Default parameters (device names)
  */
 static const candev_params_t candev_params[] = {
+#if CAN_DLL_NUMOF >= 1
```

Comment:
I don't really understand how the CAN_DLL_NUMOF is supposed to be set and how it integrate with the native can flags (is it possible to have multiple calls to --can ?).
Where is it documented btw ? Maybe we should make it more explicit that it's limited to 2 (and why 2 btw) ?

Reply:
Thanks, things are clearer now. Just found a typo in the native related README. Please squash!